cookbooks/bach_spark/templates/default/spark-env.sh.erb

```diff
@@ -0,0 +1,5 @@
+export HADOOP_CONF_DIR="/etc/hadoop/conf"
+export HDP20_BASE_DIR="/usr/lib/hadoop"
```

Comment:
No need to support HDP2.0 we have moved on to the HDP2.2 milestone for this codebase.